### PR TITLE
Add dirty code

### PR DESCRIPTION
**Pull request by pocke**

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+def foo
```
Comment (sumyapp) [inline main.rb:1]: **[rubocop]**
- Missing frozen string literal comment.

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  eval(foo)
+end
```
Comment (sumyapp) [inline main.rb:3]: **[rubocop]**
- The use of `eval` is a serious security risk.

File: main.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+class AAA
+        def initialize
```
Comment (sumyapp) [inline main.rb:6]: **[rubocop]**
- Missing top-level class documentation comment.
- Use 2 (not 8) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)

File: main.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
```
Comment (sumyapp) [inline main.rb:7]: **[rubocop]**
- Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

File: main.rb (hunk 1 above)
Comment (wata727) [inline main.rb:1]: **[staging][rubocop]**
- Missing frozen string literal comment.

File: main.rb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+def foo
+  eval(foo)
+end
+
+class AAA
+        def initialize
+          
+        end
```
Comment (sumyapp) [inline main.rb:8]: **[rubocop]**
- Extra empty line detected at method body beginning.

File: main.rb (hunk 2 above)
Comment (wata727) [inline main.rb:3]: **[staging][rubocop]**
- The use of `eval` is a serious security risk.

File: main.rb (hunk 3 above)
Comment (wata727) [inline main.rb:6]: **[staging][rubocop]**
- Missing top-level class documentation comment.
- Use 2 (not 8) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)

File: main.rb (hunk 4 above)
Comment (wata727) [inline main.rb:7]: **[staging][rubocop]**
- Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

**Comment by sumyapp** [comment]
Click here if you want to create rubocop's auto-correct Pull Request 
[Auto Correct by SideCI](https://sideci.com/gh/38860676/issues/pull_request/41/diff/0f24aca44618aa993a0eb72a2ac4c954c2f01426?open_modal=1#rubocop)

See the documentation if you want to get more information about auto-correct.
[Document](https://sideci.com/ja/docs/autofix-rubocop)

File: main.rb (hunk 5 above)
Comment (wata727) [inline main.rb:8]: **[staging][rubocop]**
- Extra empty line detected at method body beginning.

**Comment by wata727** [comment]
Click here if you want to create rubocop's auto-correct Pull Request 
[staging] 
[Auto Correct by SideCI](https://www.codemagica.com/gh/38860676/issues/pull_request/41/diff/0f24aca44618aa993a0eb72a2ac4c954c2f01426?open_modal=1#rubocop)

See the documentation if you want to get more information about auto-correct.
[Document](https://www.codemagica.com/ja/docs/autofix-rubocop)

File: main.rb [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+end
+def foo
```
Comment (wata727) [inline main.rb:10]: **[staging][rubocop]**
- Method `Object#foo` is defined at both /sideci/sideci_rails_sandbox/main.rb:2 and /sideci/sideci_rails_sandbox/main.rb:10.

File: main.rb [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+  eval(foo)
```
Comment (wata727) [inline main.rb:11]: **[staging][rubocop]**
- The use of `eval` is a serious security risk.

File: main.rb [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+end
+
+class AAA
```
Comment (wata727) [inline main.rb:14]: **[staging][rubocop]**
- Missing top-level class documentation comment.

File: main.rb (hunk 6 above)
Comment (sumyapp) [inline main.rb:10]: **[rubocop]**
- Method `Object#foo` is defined at both /sideci/sideci_rails_sandbox/main.rb:2 and /sideci/sideci_rails_sandbox/main.rb:10.

File: main.rb [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+        def initialize
```
Comment (wata727) [inline main.rb:15]: **[staging][rubocop]**
- Use 2 (not 8) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
- Method `AAA#initialize` is defined at both /sideci/sideci_rails_sandbox/main.rb:7 and /sideci/sideci_rails_sandbox/main.rb:15.

File: main.rb (hunk 7 above)
Comment (sumyapp) [inline main.rb:11]: **[rubocop]**
- The use of `eval` is a serious security risk.

File: main.rb [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+
```
Comment (wata727) [inline main.rb:16]: **[staging][rubocop]**
- Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

File: main.rb (hunk 8 above)
Comment (sumyapp) [inline main.rb:14]: **[rubocop]**
- Missing top-level class documentation comment.

File: main.rb [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+        end
+end
+def foo
+  eval(foo)
+end
+
+class AAA
+        def initialize
+          
+        end
```
Comment (wata727) [inline main.rb:17]: **[staging][rubocop]**
- Extra empty line detected at method body beginning.

File: main.rb (hunk 9 above)
Comment (sumyapp) [inline main.rb:15]: **[rubocop]**
- Use 2 (not 8) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
- Method `AAA#initialize` is defined at both /sideci/sideci_rails_sandbox/main.rb:7 and /sideci/sideci_rails_sandbox/main.rb:15.

File: main.rb (hunk 10 above)
Comment (sumyapp) [inline main.rb:16]: **[rubocop]**
- Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

**Comment by wata727** [comment]
Click here if you want to create rubocop's auto-correct Pull Request 
[staging] 
[Auto Correct by SideCI](https://www.codemagica.com/gh/38860676/issues/pull_request/41/diff/b0411ddfbbad3ae76a7724eb3df3b1450a9d8c36?open_modal=1#rubocop)

See the documentation if you want to get more information about auto-correct.
[Document](https://www.codemagica.com/ja/docs/autofix-rubocop)

File: main.rb (hunk 11 above)
Comment (sumyapp) [inline main.rb:17]: **[rubocop]**
- Extra empty line detected at method body beginning.

**Comment by sumyapp** [comment]
Click here if you want to create rubocop's auto-correct Pull Request 
[Auto Correct by SideCI](https://sideci.com/gh/38860676/issues/pull_request/41/diff/b0411ddfbbad3ae76a7724eb3df3b1450a9d8c36?open_modal=1#rubocop)

See the documentation if you want to get more information about auto-correct.
[Document](https://sideci.com/ja/docs/autofix-rubocop)

File: main.rb [hunk 12]
```diff
@@ -0,0 +1,18 @@
+def foo
```
Comment (pocke) [inline main.rb:1]: **[development][rubocop]**
- Missing frozen string literal comment.

File: main.rb [hunk 13]
```diff
@@ -0,0 +1,18 @@
...
+  eval(foo)
+end
```
Comment (pocke) [inline main.rb:3]: **[development][rubocop]**
- The use of `eval` is a serious security risk.

File: main.rb [hunk 14]
```diff
@@ -0,0 +1,18 @@
...
+
+class AAA
+        def initialize
```
Comment (pocke) [inline main.rb:6]: **[development][rubocop]**
- Missing top-level class documentation comment.
- Use 2 (not 8) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)

File: main.rb [hunk 15]
```diff
@@ -0,0 +1,18 @@
...
+
```
Comment (pocke) [inline main.rb:7]: **[development][rubocop]**
- Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

File: main.rb [hunk 16]
```diff
@@ -0,0 +1,18 @@
+def foo
+  eval(foo)
+end
+
+class AAA
+        def initialize
+          
+        end
```
Comment (pocke) [inline main.rb:8]: **[development][rubocop]**
- Extra empty line detected at method body beginning.

File: main.rb (hunk 6 above)
Comment (pocke) [inline main.rb:10]: **[development][rubocop]**
- Method `Object#foo` is defined at both /sideci/sideci_rails_sandbox/main.rb:2 and /sideci/sideci_rails_sandbox/main.rb:10.

File: main.rb (hunk 7 above)
Comment (pocke) [inline main.rb:11]: **[development][rubocop]**
- The use of `eval` is a serious security risk.

File: main.rb (hunk 8 above)
Comment (pocke) [inline main.rb:14]: **[development][rubocop]**
- Missing top-level class documentation comment.

File: main.rb (hunk 9 above)
Comment (pocke) [inline main.rb:15]: **[development][rubocop]**
- Use 2 (not 8) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
- Method `AAA#initialize` is defined at both /sideci/sideci_rails_sandbox/main.rb:7 and /sideci/sideci_rails_sandbox/main.rb:15.

File: main.rb (hunk 10 above)
Comment (pocke) [inline main.rb:16]: **[development][rubocop]**
- Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

File: main.rb (hunk 11 above)
Comment (pocke) [inline main.rb:17]: **[development][rubocop]**
- Extra empty line detected at method body beginning.

**Comment by pocke** [comment]
Click here if you want to create rubocop's auto-correct Pull Request 
[development] 
[Auto Correct by SideCI](http://localhost:3000/gh/38860676/issues/pull_request/41/diff/b0411ddfbbad3ae76a7724eb3df3b1450a9d8c36?open_modal=1#rubocop)

See the documentation if you want to get more information about auto-correct.
[Document](http://localhost:3000/ja/docs/autofix-rubocop)

File: main.rb [hunk 17]
```diff
@@ -0,0 +1,20 @@
...
+end
+
```
Comment (wata727) [inline main.rb:19]: **[staging][rubocop]**
- Line is too long. [120/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

Comment (sumyapp) [inline main.rb:19]: **[rubocop]**
- Line is too long. [120/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

**Comment by wata727** [comment]
Click here if you want to create rubocop's auto-correct Pull Request 
[staging] 
[Auto Correct by SideCI](https://www.codemagica.com/gh/38860676/issues/pull_request/41/diff/3b7c46a37a0dd7c4cf2caba7407ea74bd81a4424?open_modal=1#rubocop)

See the documentation if you want to get more information about auto-correct.
[Document](https://www.codemagica.com/ja/docs/autofix-rubocop)

**Comment by sumyapp** [comment]
Click here if you want to create rubocop's auto-correct Pull Request 
[Auto Correct by SideCI](https://sideci.com/gh/38860676/issues/pull_request/41/diff/3b7c46a37a0dd7c4cf2caba7407ea74bd81a4424?open_modal=1#rubocop)

See the documentation if you want to get more information about auto-correct.
[Document](https://sideci.com/ja/docs/autofix-rubocop)

File: main.rb (hunk 17 above)
Comment (pocke) [inline main.rb:19]: **[development][rubocop]**
- Line is too long. [120/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

**Comment by pocke** [comment]
Click here if you want to create rubocop's auto-correct Pull Request 
[development] 
[Auto Correct by SideCI](http://localhost:3000/gh/38860676/issues/pull_request/41/diff/3b7c46a37a0dd7c4cf2caba7407ea74bd81a4424?open_modal=1#rubocop)

See the documentation if you want to get more information about auto-correct.
[Document](http://localhost:3000/ja/docs/autofix-rubocop)

**Pull request closed without merging**